spec/index.html

```diff
@@ -650,20 +752,22 @@ <h2>Streams</h2>
           en: Smoke
       -->
     </pre>
-    </section>
 
     <p>
       Each of the individual <a>YAML documents</a> in the stream
-      has to be converted into a separated <a>JSON-LD document</a> and
+      is converted into a separate <a>JSON-LD document</a> and
       processed separately.
     </p>
-
-    <p class="ednote">
+    <p class="issue" data-number="63">
+      The current text does not support this, and only supports
+      a single <a>YAML document</a>.
       This is inconsistent with the processing description in <a href="#convert-stream" class="sectionRef"></a>.
```

Comment:
@gkellogg I suggest to say that:

1. ld+yaml can contain multiple documents even in the basic profile;
2. they should be processed in isolation. For further details see  https://github.com/ietf-wg-httpapi/mediatypes/blob/main/draft-ietf-httpapi-yaml-mediatypes.md#yaml-streams-int-yaml-streams 

This is because aggregating multiple docs in a single stream is a common practice and a useful native YAML feature addressed by editors and libraries.

Reply:
I'm not sure what the implications for the API would be, as it doesn't really provide for separate documents.

I think this is something we come back to later.